tests/network/network_wrappers/test_osm_network_wrapper.py

```diff
@@ -292,3 +293,27 @@ def test_given_valid_base_geometry_with_polygon(
         # 3. Verify expectations.
         assert isinstance(_wrapper, OsmNetworkWrapper)
         assert isinstance(_wrapper.polygon_graph, MultiDiGraph)
+
+    @slow_test
+    def test_given_no_output_graph_dir_when_get_network(self):
+        # 1. Define test data.
+        _test_input_directory = test_data.joinpath("graph", "test_osm_network_wrapper")
+        _polygon_file = _test_input_directory.joinpath("_test_polygon.geojson")
+        assert _polygon_file.exists()
+
+        _network_config_data = self._get_dummy_network_config_data()
+        _network_config_data.network.polygon = _polygon_file
+        _network_config_data.network.network_type = NetworkTypeEnum.DRIVE.config_value
+        _network_config_data.network.road_types = ""
```

Comment:
See comment in #306:
* Don't need `config_value`.
* Why need these 2 lines.

Reply:
I fixed this in the follow-up issue (will keep it there to avoid overlapping changes

Reply:
Resolved in the previous issue, change should appear here as well when the branch is updated.